.github/renovate.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:recommended",
+    "default:pinDigestsDisabled",
+    "mergeConfidence:all-badges",
+    "docker:disable"
+  ],
+  "assignees": [
+    "I572056"
```

Comment:
This is the public GitHub. @sven-rosenzweig you probably want to use `sven-rosenzweig` as username here.